encrypted-media/clearkey-mp4-playback-temporary.html

```diff
@@ -24,31 +27,26 @@
 
   </head>
   <body>
-    <h1 class="instructions">Description</h1>
-    <p class="instructions">
-      This test verifies successful playback for a temporary session with Clear Key and mp4/cenc content.
-    </p>
-
     <div id='log'></div>
 
     <div id='video'>
-      <video id="videoelement" width="600px"></video>
+      <video id="videoelement" width="200px"></video>
```

Comment:
Why this size change?  I guess an arbitrarily small element is fine for testing, but this seems not a typical end user size.